pages/sponsor/prospectus.i18n.js

```diff
@@ -0,0 +1,383 @@
+            'Recorded tech talks will be uploaded to several platforms, such as Youtube, ' +
+                'where logos will be shown in the opening or ending depending on the package.',
+            'Job fair introduction is restricted to 5 minutes per organization.',
+            'PyCon TW 2021 will not produce a digital brochure, all information will be posted ' +
+                'on our website and app.',
+            'PyCon TW 2021 do not provide tote bag and physical brochure to our participants. ' +
+                'However, sponsors have the liberty to design, print and provide copies of ' +
+                'their own handbook.',
+            'The sponsoring talk is limited to 30 minutes, and 1 talk per organization.',
+            'Diamond Sponsorship includes title rights, the company logo will be displayed with ' +
+                'PyCon TW 2021. However, PyCon TW 2021 reserves the right to further evaluate such action.',
+            'Diamond Sponsorship includes the title right of the main conference room, ' +
+                'PyCon TW 2021 reserves the right to evaluate such action.',
+            'To further promote diversity of PyCon TW 2021, company(group / sponsoring) tickets will not ' +
+                'exceeded 1 / 3 of the total pass sold.Thus, such group passes are sold in a first come ' +
+                'first served manner, please order ASAP.',
+            'Sponsor packages may be affected and changed due to the outbreak of COVID-19, ' +
+                'final version will be released on mid June along with conference details.',
+        ],
+
+        terms: {
+            diamond: 'Diamond Level Sponsorship',
+            platinum: 'Platinum Level Sponsorship',
+            gold: 'Gold Level Sponsorship',
+            silver: 'Silver Level Sponsorship',
+            bronze: 'Bronze Level Sponsorship',
+            special: 'Special and Customized Sponsorship',
+        },
+    },
+    'zh-hant': {
+        title: '贊助方案',
+        contact: '聯絡方式：電子信箱 {mail} 致「台灣 Python 年會贊助組」收',
+        contactWindows: [
+            '台灣 Python 年會 2021 大會主席 李唯',
+            '台灣 Python 年會 2021 議程主席 柯文華',
+            '台灣 Python 年會 2021 贊助總監 王品淳',
+        ],
+        intro: [
+            '台灣 Python 年會，是每年在台灣舉辦、並且聚集了最多的在地的使用、開發 Python 的社群技術研討會。' +
+                '台灣 Python 年會是在 Python Software Foundation 授權下，由 Python 社群與財團法人開放文化基金會' +
+                '（統一編號 38552170）合作，自發性地為了 Python 社群自身所舉辦的年度技術研討會。',
+            '台灣 Python 年會歷年來持續促進台灣的 Python 社群彼此之間，與台灣與國際上的 Python 社群彼此之間，' +
+                '有更多、更好、更緊密的交流，以期促進 Python 語言的成長與茁壯。' +
+                '您的贊助會使得台灣 Python 年會能夠持續運作，並且盡可能觸及更多的會眾前往交流。' +
+                '我們不只舉辦年會，也透過財務補助的方式，鼓勵更多人前往參與台灣 Python 年會。',
+            '我們期待您的來訊洽詢，請隨時透過上方的聯繫方式與我們聯繫、洽談最適合您的贊助方式，' +
+                '以達到您想要在 Python 社群曝光、徵才或是單純回饋 Python 社群的初衷與期待。',
+        ],
+
+        stats: [
+            '2020 快速總覽',
+            [
+                '出席人數 400+ （2020 為 PyCon TW 首次移師南台灣）',
```

Comment:
Since the parenthesis is already in full form, i'll suggest that the space before it could be removed
```suggestion
                '出席人數 400+（2020 為 PyCon TW 首次移師南台灣）',
```